data/json/mapgen/shelter.json

```
@@ -329,26 +357,40 @@
         "    `!!!!`!!!!`!!!!`    ",
         "    `!!!!`!!!!`!!!!`    ",
         "    &&&&&&&&&&&&&&&&    ",
-        "           ##:#++#:##   ",
-        "   #########%......l#   ",
-        "   #*|cccc6........l#   ",
+        "       ######:#++#:##   ",
+        "   #####cccc6......l#   ",
+        "   #*#.............l:   ",
```

```suggestion
        "   #*#.............l#   ",
```
I think putting a window there behind the lockers is a bit silly, it makes the window inaccessible and there's already a window on the other side of the corner a couple tiles away.